package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
delete this file. it is not necessary

Reply:
Yeah, I deleted the extra files in the new commit .. and also changed the alignment of the image with respect to the link section of footer.. pls can you check it out again?

Reply:
please provide a deployment link to test deployment also change the inline css to classbased css.

Reply:
deployment link i tried but .. netlify was not providing link for the branch files ... instead it was creating link for the master branch for my repo of the clone project..

Reply:
let me try again.. though i did this yesterday but still no change .. it was not taking my branches into the site link ..